debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlStreamingChangeEventSource.java

```diff
@@ -128,6 +115,7 @@
     private final EventDispatcher<TableId> eventDispatcher;
     private final MySqlOffsetContext offsetContext;
     private final ErrorHandler errorHandler;
+    private Instant eventTimestamp;
```

Comment:
Does it need to be volatile?

Reply:
No, it is used by the binlog client thread only